src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
Consider using a class for styling instead of the `body` tag directly, unless there is a specific reason to do so. Tag selectors can cause specificity issues and are not as reusable as class selectors.